modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java

```diff
@@ -26,7 +26,7 @@
 
 public class OpenIdCredentialRefreshable extends WrappingOpenIdCredential
 {
-    private static final Logger LOG =
+    private static final Logger logger =
```

Comment:
Hi, thanks for your feedback! We (lotta and I) saw different naming styles within the code. One was "LOG", one "_log" and the last one "logger". We thought that it might be good to unify these names. After some researches we thought that "logger" would be the best name because a logger reference is not a constant value and therefore uppercase is maybe not the best case :) But if you already have a certain naming convention it might be superfluous to change that. Can I make the changes directly in this pull request or should I make a new one without renaming?